tests/test_render.py

```diff
@@ -777,3 +777,55 @@ def on_key_press(self, symbol: int, modifiers: int):
     # Make sure that the result is valid
     with open(IMAGE_FILENAME, "rb") as f:
         assert f.read() == png_snapshot
+
+
+@pytest.mark.required
+@pytest.mark.parametrize(
+    "renderer_type",
+    [RENDERER_TYPE.BATCHRENDER_RASTERIZER],
+)
+def test_render_planes(tmp_path, png_snapshot, renderer):
+    # Manually run this test with different parameters to make sure they run sequentially
+    # Running in parallel may crash the test OOM.
+    sizes = [(1.5, 1.8), (10.0, 7.0)]
+    n_tiles = [(0.5, 0.75), (5.0, 3.0), None]
+    for test_idx, (size, n_tiles) in enumerate(itertools.product(sizes, n_tiles)):
+        CAM_RES = (512, 512)
+        NUM_STEPS = 1
+        scene = gs.Scene(
+            renderer=renderer,
+        )
+        plane = scene.add_entity(
+            gs.morphs.Plane(size=size, n_tiles=n_tiles),
+        )
+        camera = scene.add_camera(
+            res=CAM_RES,
+            pos=(1.5, 0.5, 1.5),
+            lookat=(0.0, 0.0, 0.5),
+            fov=45,
+            GUI=False,
+        )
+        scene.add_light(
+            pos=[0.0, 0.0, 1.5],
+            dir=[1.0, 1.0, -2.0],
+            directional=1,
+            castshadow=1,
+            cutoff=45.0,
+            intensity=0.5,
+        )
+        scene.build()
+
+        # Create image exporter
+        exporter = FrameImageExporter(tmp_path)
+
+        # Simulate
+        for i in range(NUM_STEPS):
+            # Move forward step forward in time
+            scene.step()
+            rgba, depth, _, _ = camera.render(rgb=True, depth=True)
+            exporter.export_frame_single_camera(i, test_idx, rgb=rgba, depth=depth)
+
+    # Verify that the output is correct pixel-wise over multiple simulation steps
+    for image_file in sorted(tmp_path.rglob("*.png")):
```

Comment:
Remove all the comments. This test is simple. No need to explain what is going on.